pyproject.toml

```diff
@@ -88,6 +88,11 @@ markers = [
     'rust_oldstable',
     'rust_stable',
     'tumbleweed_latest',
+    # not used, but required:
+    'bci-sle15-kernel-module-toolkit_15.3',
```

Comment:
15.3 is not in the list of available versions above, which variant is correct?

Reply:
I've resolved it by not adding marks to containers that are unavailable for this `OS_VERSION`